lib/interpret_date/date_type.rb

```diff
@@ -0,0 +1,11 @@
+require "active_record"
+
+module InterpretDate
+  class DateType < ActiveRecord::Type::Date
```

Comment:
I think it would be beneficial to update the readme to include this new type so future implementers know it can be used.

Reply:
Amended in 640e77e.